rest/api_collections_test.go

```diff
@@ -269,6 +269,125 @@ func TestMultiCollectionDCP(t *testing.T) {
 	// require.NoError(t, rt.WaitForDoc(docID))
 }
 
+func TestMultiCollectionChannelAccess(t *testing.T) {
+	base.TestRequiresCollections(t)
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
+
+	if !base.TestUseXattrs() {
+		t.Skip("Test relies on import - needs xattrs")
+	}
+
+	tb := base.GetTestBucket(t)
+	defer tb.Close()
+
+	scopesConfig := GetCollectionsConfig(t, tb, 3)
+	dataStoreNames := GetDataStoreNamesFromScopesConfig(scopesConfig)
+	c1SyncFunction := `function(doc) {access(doc.username, [doc.grant1,doc.grant2]);
```

Comment:
given the test is designed to be testing explicit/admin_channel grants, it might be worht removing this to reduce the complexity of the test, and any potential side effects of a dynamic grant occurring.

Reply:
addressed in [14b870c](https://github.com/couchbase/sync_gateway/pull/6012/commits/14b870cf30370886162344a52d2606ce581b002a)